site_scons/site_tools/st-flash.py

```diff
@@ -14,6 +14,11 @@ def generate(env):
             "st-flash write $SOURCE 0x08000000"
         )
     )
+    env['BUILDERS']['flash_dbg'] = SCons.Builder.Builder(
+        action = SCons.Action.Action(
+            "st-flash --hot-plug write $SOURCE 0x08000000"
```

Comment:
--hot-plug does not restart the MCU upon finishing FLASH-ing

Reply:
hmmmm weird, on my end the way it's implemented now already doesn't reset the MCU, I'm doing it manually as part of the gdb init script